OwinOAuthProvidersDemo/App_Start/Startup.Auth.cs

```diff
@@ -3,299 +3,305 @@
-			//	clientId: "",
-			//	clientSecret: "",
-			//	isSandbox: false);
-
-			//app.UseWargamingAccountAuthentication("", WargamingAuthenticationOptions.Region.NorthAmerica);
-
-			//app.UseFlickrAuthentication("", "");
-			//app.UseVisualStudioAuthentication(
-			//	appId: "",
-			//	appSecret: "");
-
-			//app.UseSpotifyAuthentication(
-			//    clientId: "",
-			//    clientSecret: "");
-
-			//var options = new SlackAuthenticationOptions
-			//{
-			//    ClientId = "",
-			//    ClientSecret = "",
-			//    TeamId = "" // optional
-			//};
-			//options.Scope.Add("identify");
-			//app.UseSlackAuthentication(options);
-
-			//app.UseGitterAuthentication(
-			//    clientId: "",
-			//    clientSecret: ""
-			//);
-
-			//app.UseImgurAuthentication(
-			//    new ImgurAuthenticationOptions
-			//    {
-			//        ClientId = "",
-			//        ClientSecret = ""
-			//    });
-
-			//var options = new BacklogAuthenticationOptions
-			//{
-			//    ClientId = "",
-			//    ClientSecret = "",
-			//    ContractName = "",
-			//    CallbackPath = new PathString(""),  // ex.new PathString("/OauthTokenRequest")
-			//    Provider = new BacklogAuthenticationProvider
-			//    {
-			//        OnAuthenticated = async context => await System.Threading.Tasks.Task.Run(()=> { System.Diagnostics.Debug.WriteLine(String.Format("Refresh Token: {0}", context.RefreshToken)); })
-			//    }
-			//};
-
-			//app.UseBacklogAuthentication(options);
-
-			//var cosignOptions = new CosignAuthenticationOptions
-			//{
-			//    AuthenticationType = "Cosign",
-			//    SignInAsAuthenticationType = signInAsType,
-			//    CosignServer = "weblogin.umich.edu",
-			//    CosignServicePort = 6663,
-			//    IdentityServerHostInstance = "core1",
-			//    ClientServer = "cosignservername"
-			//};
-			//app.UseCosignAuthentication(cosignOptions);
-
-			//app.UseVimeoAuthentication("", "");
-
-			//app.UseFitbitAuthentication(new FitbitAuthenticationOptions
-			//{
-			//    ClientId = "",
-			//    ClientSecret = ""
-			//});
-
-			//app.UseOnshapeAuthentication(
-			//    appKey: "",
-			//    appSecret: "");
-			//
-			//
-			//app.UseOnshapeAuthentication(new OnshapeAuthenticationOptions()
-			//{
-			//  AppKey = "",
-			//  AppSecret = "",
-			//  CallbackPath = new PathString("/oauthRedirect"),
-			//  Hostname = "partner.dev.onshape.com"
-			//});
-
-			//app.UseVKontakteAuthentication("", "");
-
-			//app.UseXingAuthentication("", "");
-
-			//app.UseDoYouBuzzAuthentication("", "");
-			//app.("", "");
-			//app.UseOrcidAuthentication("","");
-
-			//app.UseDiscordAuthentication("", "");
-			//app.UseGeniAuthentication("", "");
-			//app.UseMyHeritageAuthentication("", "");
-
-			//app.UseWSO2Authentication("", "", "");
-
-      //app.UseEvernoteAuthentication(new EvernoteAuthenticationOptions()
-		  //{
-		  //      IsSandBox = true,
-      //      AppKey = "",
-      //      AppSecret = ""
-      //});
-		}
-	}
+    public partial class Startup
```

Comment:
Certainly didn't change it on purpose. I will undo the whitespace changes once I figure out how to revert a file in Git. I'm used to Subversion.